lynxkite-graph-analytics/src/lynxkite_graph_analytics/lynxkite_ops.py

```diff
@@ -75,6 +75,45 @@ def import_file(
     return core.Bundle(dfs={table_name: df})
 
 
+@op(
+    "Export to File",
+)
+def export_to_file(
+    bundle: core.Bundle,
+    *,
+    filename: str,
+    file_format: FileFormat = FileFormat.csv,
+    columns: str = "<all>",
+    index: bool = False,
+):
+    """Exports a DataFrame to a file.
+
+    Args:
+        bundle: The bundle containing the DataFrame to export.
+        filename: The name of the file to export to.
+        file_format: The format of the file to export to. Defaults to CSV.
+        columns: Comma-separated list of columns to export.
+            Defaults to "<all>" which exports all columns.
+        index: Whether to include the DataFrame index in the exported file. Defaults to False.
+    """
+
+    df = bundle.dfs["result"]
```

Comment:
What if I want to export a different table? Please make the table name a parameter. (A string is fine. A lot of boxes already take a table name parameter. At some point I will fix them all to offer a dropdown.)